### DISCUSSION TITLE
Try which Ubuntu packages for installing ROCm on Debian

### DISCUSSION THREAD

**Discussion by christian-2** [ROCm/ROCm#2872] [Q&A] [answered]
Did anybody here have success in installing ROCm on Debian 12 or 11? I am wondering whether I should try with [packages](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/native-install/ubuntu.html) for either Ubuntu 22.04 or 20.04.

**Comment by nairboon** [comment, marked as answer]
Yes it's working, see here for a guide: [Install AMD ROCm 6 on Debian stable (Bookworm)](https://blog.moubou.com/2024/05/install-amd-rocm-6-on-debian-stable-bookworm/)
